model/shared/src/main/scala/explore/model/ObservingModeSummary.scala

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.option.*
+import clue.data.syntax.*
+import lucuma.core.enums.GmosAmpReadMode
+import lucuma.core.enums.GmosNorthFilter
+import lucuma.core.enums.GmosNorthFpu
+import lucuma.core.enums.GmosNorthGrating
+import lucuma.core.enums.GmosRoi
+import lucuma.core.enums.GmosSouthFilter
+import lucuma.core.enums.GmosSouthFpu
+import lucuma.core.enums.GmosSouthGrating
+import lucuma.core.util.Display
+import lucuma.schemas.ObservationDB.Types.GmosNorthLongSlitInput
+import lucuma.schemas.ObservationDB.Types.GmosSouthLongSlitInput
+import lucuma.schemas.ObservationDB.Types.ObservingModeInput
+import lucuma.schemas.model.CentralWavelength
+import lucuma.schemas.model.ObservingMode
+import lucuma.schemas.odb.input.*
+
+// Observing mode with explicit values merged over defaults. Used for grouping observations by configuration.
+enum ObservingModeSummary derives Eq:
+  case GmosNorthLongSlit(
+    grating:           GmosNorthGrating,
+    filter:            Option[GmosNorthFilter],
+    fpu:               GmosNorthFpu,
+    centralWavelength: CentralWavelength,
+    ampReadMode:       GmosAmpReadMode,
+    roi:               GmosRoi
+  ) extends ObservingModeSummary
+  case GmosSouthLongSlit(
+    grating:           GmosSouthGrating,
+    filter:            Option[GmosSouthFilter],
+    fpu:               GmosSouthFpu,
+    centralWavelength: CentralWavelength,
+    ampReadMode:       GmosAmpReadMode,
+    roi:               GmosRoi
+  ) extends ObservingModeSummary
+
+  def toInput: ObservingModeInput = this match
+    case GmosNorthLongSlit(grating, filter, fpu, centralWavelength, ampReadMode, roi) =>
```

Comment:
Is  this matching the grouping on the odb?
How do we keep them in sync?

Reply:
No, this is independent of the groupings in the ODB. The ODB groupings take into account all of the `ObservationMode` parameters.

Reply:
What happens when two configurations differ only by parameters that are not present in the summary?

Reply:
> What happens when two configurations differ only by parameters that are not present in the summary?

They are lumped together. Then when one of them is selected, only those parameters are applied. The rest of the parameters are not sent in the mutation. Does the ODB keep the old values in this case? Should we set them to null?

Reply:
Yes, with a mutation in general only the fields that you set are modified and the rest are left alone.  If there are fields that default based on the values in the part being modified, they may change.  That would be the case for the binning.  On the other hand, if the binning is explicitly set it will not change even if it doesn't make sense for the new instrument configuration.

Reply:
It sounds to me then that we want to reset the overrides. Does this sound right @andrewwstephens ?